multi-node/aws/cmd/kube-aws/command_validate.go

```diff
@@ -25,7 +25,12 @@ var (
 
 func init() {
 	cmdRoot.AddCommand(cmdValidate)
-	cmdValidate.Flags().BoolVar(&validateOpts.awsDebug, "aws-debug", false, "Log debug information from aws-sdk-go library")
```

Comment:
nit: I prefer oneliners for flags (more readable if there are multiple flags, so in this case not huge deal)